lib/streamlit/ReportThread.py

```diff
@@ -171,5 +181,5 @@ def get_report_ctx():
     return ctx
 
 
-# Avoid circular dependencies in Python 2
+# Needed to avoid circular dependencies.
```

Comment:
Just to verify, is this still needed to resolve circular dependencies in Python 3? Just wanted to check if you tested vs. just adjusting the comment.

Reply:
Apparently, yes.  Taking this out breaks everything.

Reply:
Error without it:

```========================================================================= ERRORS =========================================================================
________________________________________________ ERROR collecting tests/streamlit/delta_generator_test.py ________________________________________________
tests/streamlit/delta_generator_test.py:131: in <module>
    class DeltaGeneratorTest(testutil.DeltaGeneratorTestCase):
tests/streamlit/delta_generator_test.py:154: in DeltaGeneratorTest
    ???
streamlit/DeltaGenerator.py:126: in wrapped_method
    return dg._enqueue_new_element_delta(marshall_element, delta_type, last_index)
streamlit/DeltaGenerator.py:353: in _enqueue_new_element_delta
    if self._container and self._cursor:
streamlit/DeltaGenerator.py:312: in _cursor
    return cursor.get_container_cursor(self._container)
streamlit/cursor.py:22: in get_container_cursor
    ctx = get_report_ctx()
streamlit/ReportThread.py:175: in get_report_ctx
    if ctx is None and streamlit._is_running_with_streamlit:
E   NameError: name 'streamlit' is not defined
```